v2/fleet-local/control/mesos-master@.service

```diff
@@ -30,7 +32,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
   --log_dir=/var/log/mesos \
   --quorum=3 \
   --work_dir=/var/lib/mesos/master \
-  --zk=zk://`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos"
+  --zk=zk://$($zk_username):$($zk_password)@`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos"
```

Comment:
let's take this chance to move the `etcdctl get /environment/ZOOKEEPER_ENDPOINT` into an `Environment`, the same way that you have `zk_username` & `zk_password`